auth-server/src/services/email.ts

```diff
@@ -109,3 +111,40 @@ export const sendPostSubscriptionMail = (user: User, author: User, comment) => {
 	sgMail.send(msg).catch(e =>
 		console.error('Post subscription email not sent', e));
 };
+
+export const sendEmailUndoEmail = (user: User, undoToken: EmailUndoToken) => {
+	if (!apiKey) {
+		console.warn('Email undo token email not sent due to missing API key');
+		return;
+	}
+
+	const undoUrl = `${DOMAIN}/undo-email/${undoToken.token}`;
+	const text = `
+		<p>
+			Hi ${user.name || ''}!<br/><br/>
+
+			It looks like someone has changed your email at polkassembly.<br />
```

Comment:
```suggestion
			Your email on polkassembly.io was changed.<br />
```

Reply:
We need to mention the new email here, it was changed to "newEmail@bla.com"

Reply:
added new email